apps/chat/src/components/Common/ApplicationDialog.tsx

```diff
@@ -212,18 +215,33 @@ const ApplicationDialogView: React.FC<Props> = ({
     }
 
     try {
-      const object: Record<string, string> = JSON.parse(data);
+      const object = JSON.parse(data);
+
+      if (typeof object === 'object' && object !== null) {
+        for (const [key, value] of Object.entries(object)) {
+          if (!key.trim()) {
+            return t('Keys should not be empty');
+          }
+
+          if (['boolean', 'number'].includes(typeof value) || value === null) {
```

Comment:
if you execute several times method (typeof) with the same variable just save it in variable (DRY)
And execute light checking first before cycle loop
```suggestion
          const valueType = typeof value;
          if (value === null || ['boolean', 'number'].includes(valueType)) {
```